### PR TITLE
Bugfix/284 sticky does not apply to forum

### PR DESCRIPTION
**Pull request by slawa85**
Meets #284 sticky does not apply to forum

### CODE REVIEW

**Comment by hschink** [comment]
@slawa85 I cannot merge this PR. Please update your OpenProject clone and rebase this, so it can be merged.

**Comment by hschink** [comment]
@slawa85 I created 3 forum messages with _sticky_ set. If I remove _sticky_ from the first message and then set it again, the message would still appear as the first message (and not the last as expected).

I recommend to remove the cuke (integration test) and introduce some controller tests and check the content of `@topics`.

File: spec/controllers/boards_controller_spec.rb [hunk 1]
```diff
@@ -175,4 +175,63 @@
...
+        end
+
+        it "it should not be displayed as sticky message" do
+
+          expect(sticked_message1.sticked_on).to be_nil
+          expect(assigns[:topics][0].id).to_not eq(sticked_message1.id)
+        end
+      end
+
+      describe "when sticky is set back to message" do
```
Comment (hschink) [inline spec/controllers/boards_controller_spec.rb:220]: @slawa85 this misses the case when `sticked_message2`'s sticky date is reset and the order changes.

**Comment by slawa85** [comment]
- `#284` Fix: Sticky does not apply to forum

**Comment by hschink** [comment]
Cleaned up and [merged](https://github.com/opf/openproject/commit/05b47774def181e57ec2a4b58324b7dc3f9b4523).

**Pull request closed without merging**